app/build.gradle

```diff
@@ -150,78 +150,58 @@ android {
         textReport true
     }
     namespace 'de.digitalService.useID'
-    applicationVariants.all { variant ->
```

Comment:
Configuration of source sets for KSP generated code is not needed any more with latest KSP plugin